server/index.js

```diff
@@ -41,9 +41,14 @@ app.use('/graphql', graphqlHTTP({
     graphiql: true
 }));
 
+// React router takes care of routing, so all endpoints should arrive on the same 'page'.
+app.get('*', (req, res) => {
```

Comment:
Since this is a wildcard, we should keep this at the end of the file, below all `app.get`'s